internal/manager/config.go

```diff
@@ -196,6 +196,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.ProbeAddr, "health-probe-bind-address", fmt.Sprintf(":%v", HealthzPort), "The address the probe endpoint binds to.")
 	flagSet.Float32Var(&c.ProxySyncSeconds, "proxy-sync-seconds", dataplane.DefaultSyncSeconds,
 		"Define the rate (in seconds) in which configuration updates will be applied to the Kong Admin API.")
+	flagSet.DurationVar(&c.InitCacheSyncDuration, "init-cache-sync-duration", 5*time.Second, `The initial delay to wait for Kubernetes object caches to be synced before the initial configuration`)
```

Comment:
I would use a const value instead of a hardcoded value for the default, as it is the approach we have for all the flags.